packages/amplication-data-service-generator-runner/src/utils/DynamicPackageInstallationManager.ts

```diff
@@ -0,0 +1,27 @@
+import { valid } from "semver";
+import { Tarball } from "./Tarball";
+export class DynamicPackageInstallationManager {
+  constructor(private pluginInstallationPath: string) {}
+
+  async installMany(packages: PackageInstallation[]): Promise<void> {
+    for (const plugin of packages) {
+      await this.install(plugin);
```

Comment:
any fail here will break the loop. you should manage it with generator or outer function with try/catch

Reply:
We want the process to crash if not installed correctly